epub33/core/index.html

```diff
@@ -9099,10 +9099,10 @@ <h2>Accessibility</h2>
 				to <a>EPUB Publications</a>. It also adds EPUB-specific requirements and recommendations for metadata,
 				pagination, and media overlays.</p>
 
-			<p>This specification recommends that EPUB Publications <a href="#confreq-a11y">conform to the EPUB
-					Accessibility standard</a>. A benefit of following this recommendation is that it helps ensure that
-				EPUB Publications meet the accessibility requirements legislated in jurisdictions around the world,
-				ensuring EPUB Creators are not locked out of potential markets.</p>
+			<p>This specification <a href="#confreq-a11y">recommends</a> that EPUB Publications conform to the 
+				accessibility requirements defined in [[EPUB-A11Y-11]]. A benefit of following this recommendation is 
```

Comment:
Better to put the link on the context of where it goes, otherwise you reach the link before you know why it's there.
```suggestion
			<p>This specification recommends that EPUB Publications <a href="#confreq-a11y">conform to the 
				accessibility requirements</a> defined in [[EPUB-A11Y-11]]. A benefit of following this recommendation is 
```

Reply:
I can go either way.